cypress/integration/users/loans/manual-anonymization-closed-loans.spec.js

```diff
@@ -0,0 +1,173 @@
+
+describe('ui-users-loans: Manual anonymization in closed loans', () => {
+  const loanTypeName = `autotest_loan_type${getRandomPostfix()}`;
+  const newOwnerData = getNewOwner();
+  const newFirstItemData = getNewItem();
+  const newSecondItemData = getNewItem();
+  const feeFineType = uuid();
+  let servicePointId;
+  let servicePoints = [];
+
+  beforeEach(() => {
+    let source;
+
+    cy.getAdminToken();
+    cy.getMaterialTypes({ limit: 1 });
+    cy.getInstanceTypes({ limit: 1 });
+    cy.getLocations({ limit: 1 });
+    cy.getHoldingTypes({ limit: 1 });
+
+    cy.then(() => {
+      ServicePoints.getViaApi()
+        .then((res) => {
+          servicePointId = res[0].id;
+          servicePoints = res;
+        });
+
+      cy.createLoanType({ name: loanTypeName });
+
+      RequestPolicy.createApi();
+      LostItemFeePolicy.createApi();
+      OverdueFinePolicy.createApi();
+      NoticePolicy.createApi();
+      cy.createLoanPolicy({
+        loanable: true,
+        loansPolicy: {
+          closedLibraryDueDateManagementId: 'CURRENT_DUE_DATE_TIME',
+          period: {
+            duration: 5,
+            intervalId: 'Minutes'
+          },
+          profileId: 'Rolling',
+        },
+        renewable: true,
+        renewalsPolicy: {
+          numberAllowed: 0,
+          renewFromId: 'CURRENT_DUE_DATE',
+        },
+      }).then(() => {
+        const loanPolicy = Cypress.env(CY_ENV.LOAN_POLICY).id;
+        const requestPolicyId = Cypress.env(CY_ENV.REQUEST_POLICY).id;
+        const noticePolicyId = Cypress.env(CY_ENV.NOTICE_POLICY).id;
+        const overdueFinePolicyId = Cypress.env(CY_ENV.OVERDUE_FINE_POLICY).id;
+        const lostItemFeesPolicyId = Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY).id;
+        const materialTypeId = Cypress.env('materialTypes').id;
+        const policy = `l ${loanPolicy} r ${requestPolicyId} n ${noticePolicyId} o ${overdueFinePolicyId} i ${lostItemFeesPolicyId}`;
+        const priority = 'priority: number-of-criteria, criterium (t, s, c, b, a, m, g), last-line';
+        const newRule = `${priority}\nfallback-policy: ${policy}\nm ${materialTypeId}: ${policy}`;
+
+        cy.updateCirculationRules({
+          rulesAsText: newRule,
+        });
+      });
+      source = InventoryHoldings.getHoldingSources({ limit: 1 });
+    }).then(() => {
+      cy.createTempUser([
+        permissions.uiUsersViewLoans.gui,
+        permissions.uiUsersDeclareItemLost.gui,
+        permissions.uiUserLoansAnonymize.gui,
+        permissions.uiFeeFines.gui,
+      ]).then(({
+        username,
+        password,
+        userId,
+        barcode: userBarcode,
+      }) => {
+        UserEdit.addServicePointViaApi(servicePointId, userId).then(() => {
+          const servicePointOwner = servicePoints.map(({
+            id,
+            name,
+          }) => ({
+            value: id,
+            label: name,
+          }));
+
+          cy.createInstance({
```

Comment:
please use createFolioInstanceViaApi instead of createInstance function

Reply:
Done!